torch/quantization/quantize.py

```diff
@@ -227,6 +227,7 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     torch.nn.Conv2d: nnq.Conv2d,
     QuantStub: nnq.Quantize,
     DeQuantStub: nnq.DeQuantize,
+    nn.Identity: nnq.Identity,
```

Comment:
Do we need this? We need not add nn.Identity to the list of modules to swap. nn.Identity already works for quantized tensors.

Reply:
oh, reason is we need to remove observer

Reply:
maybe avoid adding observer for Identity?

Reply:
Yes, I would recommend that. We need a list of modules for which we will not insert observers eventually.